### PR TITLE
[release/v2.22] Update dependencies in web-termial docker image and bump util version

### PR DESCRIPTION
**Pull request by pkprzekwas**
**What this PR does / why we need it**:
 Update dependencies in web-termial docker image and bump util version

**What type of PR is this?**
/kind chore

**Does this PR introduce a user-facing change? Then add your Release Note here**:

```release-note
Update web-terminal image to kubectl 1.25.14, Helm 3.12.3 and curl 8.4.0
```

**Documentation**:

```documentation
NONE
```

### CODE REVIEW

File: hack/images/web-terminal/release.sh [hunk 1]
```diff
@@ -20,7 +20,7 @@ cd $(dirname $0)/../../..
 source hack/lib.sh
 
 REPOSITORY=quay.io/kubermatic/web-terminal
-VERSION=0.6.0
+VERSION=0.6.1
```
Comment (ahmedwaleedmalik) [inline hack/images/web-terminal/release.sh:23]: `0.6.1` already exists against v2.23, https://github.com/kubermatic/dashboard/blob/release/v2.23/hack/images/web-terminal/release.sh#L23. You'll need to create a different tag.

File: hack/lib.sh [hunk 2]
```diff
@@ -109,7 +109,7 @@ is_containerized() {
 
 containerize() {
   local cmd="$1"
-  local image="${CONTAINERIZE_IMAGE:-quay.io/kubermatic/util:2.2.0}"
+  local image="${CONTAINERIZE_IMAGE:-quay.io/kubermatic/util:2.3.1}"
```
Comment (ahmedwaleedmalik) [inline hack/lib.sh:112]: Why do we need this change?

**Review by ahmedwaleedmalik — CHANGES_REQUESTED** [review]
The image tag needs to be updated here as well https://github.com/kubermatic/dashboard/blob/release/v2.22/modules/api/pkg/handler/websocket/terminal.go#L68

File: hack/lib.sh (hunk 2 above)
Comment (pkprzekwas) [reply hack/lib.sh:112]: Correct

Comment (pkprzekwas) [reply hack/lib.sh:112]: https://github.com/kubermatic/kubermatic/pull/12694

**Review by embik — APPROVED** [review]

**Pull request merged**